### DISCUSSION TITLE
Set a window to be always on top?

### DISCUSSION THREAD

**Discussion by jftjamf** [rxhanson/RectanglePro-Community#585] [Q&A] [answered]
Can either Rectangle or Rectangle Pro pin a window such that it is `always on top`, but not necessarily focused? 

Scenario: I want to switch between PyCharm and Terminal via `command-tab`. When Terminal is activated, I'd still like to see PyCharm fully "in front (or on top)", even though Terminal is active.

If this is possible, what is the procedure? If not, are there other alternatives that can accomplish this?

Thanks.

**Comment by rxhanson** [comment, marked as answer]
Unfortunately, macOS doesn't have a supported way to do this. Rectangle Pro can make its own windows stay on top (i.e. the Preferences window), but it cannot make windows belonging to other applications stay on top. 

The only way to achieve something like this currently is to float a window that shows a screen capture of another window, and this isn't something I have plans to add in at the moment.

In Rectangle Pro, there is Pin mode, but that is for reflowing windows around a window, not for floating a window.

**Reply by jftjamf** [reply]
Thanks for your detailed response. I really appreciate it. 😀

**Comment by luckman212** [comment]
Yeah that's how https://github.com/lihaoyun6/Topit does it (ScreenCaptureKit). It's pretty fast/efficient though (and useful!). Hope Rectangle Pro might get this feature one day.
